low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -128,6 +129,35 @@ def get_controllers_info(self):
             data.append(controller_dict)
         return data
 
+    def get_psus_info(self):
```

Comment:
lets just keep get_psu_info() name, no plural as its confusing without the capital PSU in place